superset/assets/src/dashboard/reducers/dashboardState.js

```diff
@@ -30,10 +30,18 @@ import {
   TOGGLE_EXPAND_SLICE,
   TOGGLE_FAVE_STAR,
   UPDATE_CSS,
+  ON_RECONCILE,
+  ON_FINISH_RECONCILE,
 } from '../actions/dashboardState';
 
 export default function dashboardStateReducer(state = {}, action) {
   const actionHandlers = {
+    [ON_RECONCILE]() {
+      return { ...state, doReconcile: true };
+    },
+    [ON_FINISH_RECONCILE]() {
```

Comment:
we can use replace word **ON_FINISH_RECONCILE** Finish word with success. Success key word looks good.

Reply:
renamed to ON_SUCCESS_RECONCILE